### PR TITLE
[Filebeat] sync checkpoint module with Fleet integration

### PR DESCRIPTION
**Pull request by andrewkroh**
## What does this PR do?

This adds `network.transport` to events derived from the IANA number. It brings in the change related to https://github.com/elastic/integrations/issues/2463.

The Filebeat module was missing definitions for these four fields.

- checkpoint.comment
- checkpoint.conn_direction
- checkpoint.db_ver
- checkpoint.update_status

On difference between Fleet and Filebeat is that the Filebeat pipeline is setting `client` and `server` based on the `source` and `destination`. That behavior was kept. Another difference is `event.ingested` is required in the Filebeat version of the pipeline. Otherwise the pipelines are the same.

This commit was used:

https://github.com/elastic/integrations/commit/2aee5ee4b2774ac653a7568c74674435f1969c05

Filebeat had some ~10000 test log lines, but only the first 100 are checked in test_modules.py so I removed 9900 of them.

## Why is it important?

Consistency is nice.

## Checklist

- [x] My code follows the style guidelines of this project
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have made corresponding change to the default configuration files
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added an entry in `CHANGELOG.next.asciidoc` or `CHANGELOG-developer.next.asciidoc`.

## Note to reviewers

I think it would be easier to review the diff commit by commit. And turn off whitespace for the pipeline YAML diff.

## Related issues

- Relates https://github.com/elastic/integrations/issues/2463
- Requires https://github.com/elastic/integrations/pull/2895

### CODE REVIEW

**Comment by andrewkroh** [comment]
At this point the failures appear to be caused by to the fact that Beats now require 8.3.0 ES versions for testing purposes (due to `output.elasticsearch.allow_older_versions: false`). So until there is an 8.3.0-SNAPSHOT build AND beats is updated to use it, then this will not pass.

**Comment by andrewkroh** [comment]
@Mergifyio update

**Review by efd6 — APPROVED** [review]

File: x-pack/filebeat/module/checkpoint/firewall/config/firewall.yml [hunk 1]
```diff
@@ -25,10 +25,6 @@ publisher_pipeline.disable_host: {{ inList .tags "forwarded" }}
 
 processors:
   - add_locale: ~
-  - add_fields:
-      target: ''
-      fields:
-        ecs.version: 1.12.0
```
[the remark this answers is not in the document]
Comment (andrewkroh) [reply x-pack/filebeat/module/checkpoint/firewall/config/firewall.yml:31]: It added ecs.version to the outgoing event. This is now being set as part of the ingest pipeline with a `set` processor.

File: x-pack/filebeat/module/checkpoint/fields.go [hunk 2]
```diff
@@ -19,5 +19,5 @@ func init() {
 // AssetCheckpoint returns asset data.
 // This is the base64 encoded zlib format compressed contents of module/checkpoint.
 func AssetCheckpoint() string {
```
[the remark this answers is not in the document]
Comment (andrewkroh) [reply x-pack/filebeat/module/checkpoint/fields.go:21]: Agree.

**Pull request merged**